src/app/Processor/ProcessorEdit/ConfigurationEdit/ConfigurationEdit.tsx

```diff
@@ -200,21 +145,16 @@ const ConfigurationEdit = (props: ConfigurationEditProps): JSX.Element => {
           />
         </FormGroup>
       )}
-      {type !== "" && schema && !schemaLoading && (
+      {type !== "" && schema && (
         <ConfigurationForm
           configuration={parameters}
           schema={schema}
           onChange={updateConfiguration}
           registerValidation={registerValidateParameters}
           readOnly={readOnly}
+          editMode={editMode}
```

Comment:
We need to know if we are creating a new processor or editing an existing one. The handling of secret values will depend on it.